content/docs/pipelines/upgrade.md

```diff
@@ -102,7 +102,7 @@ The steps below assume that you already have a Kubernetes cluster set up.
 
 ## Upgrading your Kubeflow Pipelines deployment
 
-To upgrade your Kubeflow Pipelines deployment, run the following script in the 
+To upgrade your Kubeflow Pipelines deployment to latest version, run the following script in the 
```

Comment:
s/to latest version/to the latest version

Reply:
done